Kickstarter-iOS/Views/Controllers/BackerDashboardViewControllerTests.swift

```diff
@@ -0,0 +1,23 @@
+@testable import Kickstarter_Framework
+import XCTest
+
+internal final class BackerDashboardViewControllerTests: TestCase {
+  func testScrolToTop() {
+    let timeout = 0.1
+    let vc = BackerDashboardViewController.instantiate()
+    _ = vc.view
+
+    let configureExpectation = expectation(description: "Waiting for view controllers to be configured")
+
+    DispatchQueue.main.asyncAfter(deadline: .now() + timeout) {
```

Comment:
We have to use `asyncAfter` in order for the controllers (of the page view controller) to be configured (this is due to the fact that we're A) using `.observeForControllerAction()` and B) using `DispatchQueue.main.async`